go-controller/pkg/util/multi_network_test.go

```diff
@@ -425,56 +460,83 @@ func TestIsPrimaryNetwork(t *testing.T) {
 
 	tests := []testConfig{
 		{
-			desc: "defaultNetInfo with primary unspecified",
+			desc: "defaultNetInfo with role unspecified",
+			inputNetConf: &ovncnitypes.NetConf{
+				NetConf:  cnitypes.NetConf{Name: ovntypes.DefaultNetworkName},
+				Topology: ovntypes.Layer3Topology,
+			},
+			expectedPrimary: false,
+		},
+		{
+			desc: "defaultNetInfo with role set to primary",
 			inputNetConf: &ovncnitypes.NetConf{
 				NetConf:  cnitypes.NetConf{Name: ovntypes.DefaultNetworkName},
 				Topology: ovntypes.Layer3Topology,
+				Role:     ovntypes.NetworkRolePrimary,
 			},
 			expectedPrimary: false,
 		},
 		{
-			desc: "defaultNetInfo with primary set to true",
+			desc: "defaultNetInfo with role set to secondary",
 			inputNetConf: &ovncnitypes.NetConf{
-				NetConf:        cnitypes.NetConf{Name: ovntypes.DefaultNetworkName},
-				Topology:       ovntypes.Layer3Topology,
-				PrimaryNetwork: true,
+				NetConf:  cnitypes.NetConf{Name: ovntypes.DefaultNetworkName},
+				Topology: ovntypes.Layer3Topology,
+				Role:     ovntypes.NetworkRoleSecondary,
```

Comment:
Let's use "ovntypes" values at the other tests too.

Reply:
I think I have used it in other tests as well, the only bit where I didn't use is when we directly specify the config..